source/administration/important-upgrade-notes.rst

```diff
@@ -7,6 +7,30 @@ Important Upgrade Notes
+|                                                    | in advance of the upgrade. Since this migration happens before the Mattermost Server is fully launched, non-High Availability installs will be unreachable       |
+|                                                    | during this time.                                                                                                                                                |          
+|                                                    |                                                                                                                                                                  |
+|                                                    | The migration is a single line of SQL and can be applied directly to the database through the MySQL/PSQL command line clients if you prefer to decouple this     |
+|                                                    | from restarting the Mattermost server. It is fully backwards compatible so the schema change can be applied to any previous version of Mattermost without issue. |
+|                                                    | During the time the schema change is running (~30s per million rows in the Reactions table), if end users attempt to react to posts, the emoji reactions will    | 
+|                                                    | not load for end users.                                                                                                                                          |
+|                                                    |                                                                                                                                                                  |
+|                                                    | MySQL: ``ALTER TABLE Reactions DROP PRIMARY KEY, ADD PRIMARY KEY (PostId, UserId, EmojiName);``                                                                  |
+|                                                    |                                                                                                                                                                  |
+|                                                    | Postgres: ``ALTER TABLE reactions DROP CONSTRAINT reactions_pkey, ADD PRIMARY KEY (PostId, UserId, EmojiName);``                                                 |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+                                                  
+|                                                    | The at-mentioning an LDAP group (E20) feature is not yet supported on mobile apps. In mobile apps, users who attempt to use the feature will see an error.       |
```

Comment:
@srkgupta @thefactremains I'm adding release note here about group mentions (E20) feature not supported on mobile apps. Will need more details here on what customers may experience on the mobile apps since the feature is missing.

Reply:
Hi @amyblais 
The following statement needs to be changed as its not true and users will not see any errors:
`In mobile apps, users who attempt to use the feature will see an error.`

Instead it should be reworded to something like this:
`In mobile apps, users will not be able to see the group mentions in the autocomplete dropdown. Users will still receive notifications if they are part of any group mentioned. However the group mention keyword would not be highlighted. `